### PR TITLE
Scripts arguments

### PR DESCRIPTION
**Pull request by CatoLeanTruetschel**

### CODE REVIEW

File: build/parse-args.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        }
+      }
+    }
+    
+    if(!argProcessed && args) {
+      for (const arg of args) {
+        if ('--' + arg.name === key) {
+          i++;
+
+          if(i >= input.length) {
```
Comment (CatoLeanTruetschel) [inline build/parse-args.js:39]: Missing space

File: build/parse-args.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            console.log("Found flag '" + flag.name + "'. Writing property '" + camelizedName + "' with value of true to args object.");
+          }
+
+          result[camelizedName] = true;
+          argProcessed = true;
+        }
+      }
+    }
+    
+    if(!argProcessed && args) {
```
Comment (CatoLeanTruetschel) [inline build/parse-args.js:34]: Missing space

File: build/parse-args.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+          if(i >= input.length) {
+            throw new Error('No value present for command-line argument:' + key);
+          }
+
+          const value = input[i];
+          const camelizedName = camelize(arg.name);
+
+          if (debug) {
+            console.log("Found argument '" + arg.name + "'. Writing property '" + camelizedName + "' with value '" + value +"' to args object.");
```
Comment (CatoLeanTruetschel) [inline build/parse-args.js:47]: Refactor console output formatting to dedicated function

File: build/parse-args.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+          }
+
+          result[camelizedName] = value;
+          argProcessed = true;
+        }
+      }
+    }
+
+    if(!argProcessed) {
```
Comment (CatoLeanTruetschel) [inline build/parse-args.js:56]: Missing space

**Comment by CatoLeanTruetschel** [comment]
Apply to `start-frontend` and `start-backend` scripts, too. Then it's possible to add a script to start all.

File: build/parse-args.js (hunk 3 above)
Comment (joluj) [reply build/parse-args.js:47]: It's ok IMO, we don't need to overengineer such a little script

File: build/parse-args.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      throw new Error('Unknown command-line argument: ' + key);
+    }
+  }
+
+  return result;
+}
+
+exports.parseArgs = parseArgs;
```
Comment (joluj) [inline build/parse-args.js:64]: No new line

**Review by joluj — APPROVED** [review]
LGTM

File: build/parse-args.js (hunk 1 above)
Comment (CatoLeanTruetschel) [reply build/parse-args.js]: Done!

File: build/parse-args.js (hunk 2 above)
Comment (CatoLeanTruetschel) [reply build/parse-args.js]: Done!

File: build/parse-args.js (hunk 4 above)
Comment (CatoLeanTruetschel) [reply build/parse-args.js]: Done!

File: build/parse-args.js (hunk 3 above)
Comment (CatoLeanTruetschel) [reply build/parse-args.js]: >Refactor console output formatting to dedicated function  

Done!

File: build/parse-args.js (hunk 5 above)
Comment (CatoLeanTruetschel) [reply build/parse-args.js]: Done!

**Pull request merged**